src/main.ts

```diff
@@ -450,3 +450,14 @@ dropDownElements.forEach((dropDown) => {
 });
 
 showResult(null, null);
+
+
+// Disable file opening in browser
+
+const events = ["dragover", "drop"];
+
+for ( let event of events) {
+  document.addEventListener(event, (e) => {
+    e.preventDefault();
+  })
+}
```

Comment:
code works but with the way the project is done, i personally don't like this, if you noticed(i'm sure you didn't anyway) all variables are at the top, all for each listeners are in the same place and all normal listeners are the same way. So please refactor this to fix that

Reply:
> code works but with the way the project is done, i personally don't like this, if you noticed(i'm sure you didn't anyway) all variables are at the top, all for each listeners are in the same place and all normal listeners are the same way. So please refactor this to fix that

How about these new changes?